.github/workflows/ci.yml

```diff
@@ -374,7 +374,6 @@ jobs:
   release:
     name: Release
     if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')
-    needs: build-test
```

Comment:
This line is needs to be restored. Otherwise, we may release a version that does not build properly and thus, can not be installed by users.